modules/secrets-store-sync-secrets.adoc

```diff
@@ -0,0 +1,80 @@
+
+[WARNING]
+====
+Do not enable synchronization if you do not want to store your secrets on your {product-title} cluster and in etcd. Enable this functionality only if you require it; for example, to use environment variables to refer to the secret.
+====
+
+// TODO: If they enable sync, should we recommend they enable auto rotation too, or can they be done independent of each other?
+
+If synchronization is enabled, the secrets from the mounted volume are synchronized as Kubernetes secrets after you start a pod that mounts the secrets.
+
+The synchronized Kubernetes secret is deleted when all pods consuming the secret are deleted.
+
+// TODO: This came from upstream's: "When all the pods consuming the secret are deleted, the Kubernetes secret is also deleted."
+// What exactly does "consuming it" mean? Does it mean when the pod(s) that mounted the content are deleted, or does it also mean if there are pods out there that are referencing the Kubernetes secret via ENV VAR?
+
+.Prerequisites
+
+* You have installed the {secrets-store-operator}.
+* You have installed a secrets store provider.
+* You have created the secret provider class.
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Edit the `SecretProviderClass` resource by running the following command:
++
+[source,terminal]
+----
+$ oc edit secretproviderclass my-azure-provider <1>
+----
+<1> Replace `my-azure-provider` with the name of your secret provider class.
+
+. Add the `secretsObjects` section with the configuration for the synchronized Kubernetes secrets:
++
+[source,yaml]
+----
+
+apiVersion: secrets-store.csi.x-k8s.io/v1
+kind: SecretProviderClass
+metadata:
+  name: my-azure-provider
+spec:
+  provider: azure
+  parameters:
+    keyvaultName: "kvname"
+    objects: |
+      array:
+        - |
+          objectName: tls-cert
+          objectType: secret
+    tenantId: "tid"
+  secretObjects:                          <1>
+    - secretName: ingress-tls-csi         <2>
+      type: kubernetes.io/tls             <3>
+      data:
+        - objectName: tls-cert            <4>
+          key: tls.key                    <5>
+        - objectName: tls-cert
+          key: tls.crt
+----
+<1> Specify the configuration for synchronized Kubernetes secrets.
+<2> Specify the name of the Kubernetes `Secret` object to create.
+<3> Specify the type of Kubernetes `Secret` object to create. For example, `Opaque` or `kubernetes.io/tls`.
+<4> Specify the object name or alias of the mounted content to synchronize.
+<5> Specify the data field to from the specified `objectName` to populate the Kubernetes secret with.
++
+// TODO: Check indentation - should everything starting with "secretName" be indented left 2 spaces?
```

Comment:
@ropatil010 @knelasevero

Is this indented properly, or should everything starting at the line with callout <2> be shifted left 2 spaces?

Reply:
@ropatil010 @knelasevero bump on this. Is the indentation okay here, or do we need to make any adjustments?

Reply:
yeah, identation looks good to me. Thanks.!